main/reference/zoe-api/user-seat.md

```diff
@@ -99,6 +121,7 @@ Returns a **Promise** for a number which indicates the result of the exited **Pr
 | 0 | The user didn't get what they wanted from the **Proposal**, so their offer was refunded. |
 | 1 | The user got what they wanted from the **Proposal**, so their offer is spent & gone. |
 
+This promise will be resolved when the seat exits.
```

Comment:
```suggestion
This promise will be resolved promptly once the seat exits.
```

Is technically more precise, but the original text is fine if this precision is unneeded.